cmake/globalmacros.cmake

```diff
@@ -562,3 +562,22 @@ function(ivw_add_external_projects)
         add_subdirectory(${project_root_path} ${CMAKE_CURRENT_BINARY_DIR}/ext_${FOLDER_NAME})
     endforeach()
 endfunction()
+
+#--------------------------------------------------------------------
+# Get the list of targets the given target depends on. 
+function(ivw_get_target_depending_on retval target alsoInterfaceTargets)
```

Comment:
Why not use ivw_get_target_property_recursive
It this function is needed it should be in the globalutils.cmake file...